packages/jest-console/types/index.d.ts

```diff
@@ -0,0 +1,12 @@
+declare namespace jest {
+	interface Matchers< R > {
+		toHaveErrored(): R;
+		toHaveErroredWith( ...data: any[] ): R;
+		toHaveInformed(): R;
+		toHaveInformedWith( ...data: any[] ): R;
+		toHaveLogged(): R;
+		toHaveLoggedWith( ...data: any[] ): R;
+		toHaveWarned(): R;
+		toHaveWarnedWith( ...data: any[] ): R;
+	}
```

Comment:
It would be nice to include JSDoc descriptions for these. [This package is on definitely typed so this section could easily be moved over from there.](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/wordpress__jest-console/index.d.ts)

Once this is released, we can [remove the package from definitely typed](https://github.com/DefinitelyTyped/DefinitelyTyped#removing-a-package) as it will no longer be necessary.

Reply:
Oh nice, didn't see that. I'll copy everything over from DT.